lib/Dialect/FIRRTL/Transforms/LowerLayers.cpp

```diff
@@ -257,6 +257,10 @@ class LowerLayersPass
   void preprocessModule(CircuitNamespace &ns, InstanceGraphNode *node,
                         FModuleOp module);
 
+  /// Get the verilog name of an extmodule, which could be recorded in its
+  /// defname property.
+  StringRef getExtModuleName(FExtModuleOp extModule);
```

Comment:
This feels like a missing method on `FExtModuleOp` itself. Any idea if this shows up elsewhere in the code base which would definitely warrant raising this to the class?